internal/services/netapp/netapp_pool_data_source.go

```diff
@@ -84,9 +89,12 @@ func dataSourceNetAppPoolRead(d *pluginsdk.ResourceData, meta interface{}) error
 
 	if model := resp.Model; model != nil {
 		d.Set("location", location.NormalizeNilable(&model.Location))
```

Comment:
While it didn't change in this PR, should we change this to `location.Normalize` rather than passing the address of location?

```suggestion
		d.Set("location", location.Normalize(model.Location))
```

Reply:
Good shout!